### PR TITLE
Corrected requirements for ISO install.

### PR DESCRIPTION
**Pull request by jgu17**
Moved the base software section to the clone repository section so it's
clear to the user that ISO or rpm install does not require manual
installation of the requirements.

### CODE REVIEW

File: doc/source/deployment/setup-deployer.rst [hunk 1]
```diff
@@ -87,7 +87,20 @@ There are several ways to install the SUSE Containerized OpenStack software.
    b. sudo zypper install socok8s (installs to /usr/share/socok8s)
```
Comment (jgu17) [inline doc/source/deployment/setup-deployer.rst]: @gracetai5 could you post the steps how to add the iso to the repo list etc. before the zipper install?

Comment (kantarun) [reply doc/source/deployment/setup-deployer.rst]: Is it going to be same as adding repo ?

`zypper ar <iso-image-path> socok8s_iso` 

`zypper ar openSUSE-Addon-socok8s-x86_64-Media.iso socok8s_iso`

Comment (gracetai5) [reply doc/source/deployment/setup-deployer.rst]: I copied it from yog's note, cmd similar like this:
sudo zypper addrepo iso:/?iso=/home/ardana/openSUSE-Addon-socok8s-x86_64-Build7.2-Media.iso socok8s

File: doc/source/deployment/setup-deployer.rst [hunk 2]
```diff
@@ -87,7 +87,20 @@ There are several ways to install the SUSE Containerized OpenStack software.
...
```
Comment (gracetai5) [inline doc/source/deployment/setup-deployer.rst]: @jgu17 if we add the required sw in this section, should we remove the previous section:
https://github.com/SUSE-Cloud/socok8s/blob/master/doc/source/deployment/setup-deployer.rst#base-software

File: doc/source/deployment/setup-deployer.rst (hunk 1 above)
Comment (gracetai5) [reply doc/source/deployment/setup-deployer.rst]: I copied it from yog's note, cmd similar like this:
sudo zypper addrepo iso:/?iso=/home/ardana/openSUSE-Addon-socok8s-x86_64-Build7.2-Media.iso socok8s

File: doc/source/deployment/setup-deployer.rst (hunk 2 above)
Comment (jgu17) [reply doc/source/deployment/setup-deployer.rst]: yes. good catch. Fixed

File: doc/source/deployment/setup-deployer.rst [hunk 3]
```diff
@@ -87,7 +75,23 @@ There are several ways to install the SUSE Containerized OpenStack software.
...
+     * python3-virtualenv
+     * python3-jmespath
+     * python3-netaddr
+     * python3-openstacksdk
+     * python3-openstackclient
+     * python3-heatclient
+     * which
+
+   After the required packages are installed, clone
+   `socok8s GitHub repository <https://github.com/SUSE-Cloud/socok8s>`.
```
Comment (jsuchome) [inline doc/source/deployment/setup-deployer.rst]: All these requirements can be installed either as rpm packages, as well as using `pip` (with the appropriate env variable set for socok8s that forces creating virtual env). I believe it should be mentioned here as well

Comment (jgu17) [reply doc/source/deployment/setup-deployer.rst]: fixed

**Pull request merged**